de.bmarwell.jdyninwx.app/pom.xml

```diff
@@ -0,0 +1,171 @@
+      <artifactId>de.bmarwell.jdyninwx.lib</artifactId>
+      <version>1.0.0-SNAPSHOT</version>
+    </dependency>
+
+    <dependency>
+      <groupId>info.picocli</groupId>
+      <artifactId>picocli</artifactId>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <annotationProcessorPaths>
+            <path>
+              <groupId>info.picocli</groupId>
+              <artifactId>picocli-codegen</artifactId>
+              <version>${dependency.picocli-codegen.version}</version>
+            </path>
+          </annotationProcessorPaths>
+          <compilerArgs>
+            <arg>-Aproject=${project.groupId}/${project.artifactId}</arg>
+          </compilerArgs>
+        </configuration>
+      </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+        <configuration>
+          <archive>
+            <manifest>
+              <mainClass>${project.java.mainClass}</mainClass>
+            </manifest>
+          </archive>
+        </configuration>
+      </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>copy-runtime-dependencies</id>
+            <goals>
+              <goal>copy-dependencies</goal>
+            </goals>
+            <phase>prepare-package</phase>
+            <configuration>
+              <excludeScope>test</excludeScope>
+              <includeScope>runtime</includeScope>
+              <prependGroupId>true</prependGroupId>
+              <useSubDirectoryPerScope>true</useSubDirectoryPerScope>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+
+      <plugin>
+        <groupId>org.jreleaser</groupId>
+        <artifactId>jreleaser-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>default-assemble</id>
+            <phase>package</phase>
+            <goals>
+              <goal>assemble</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <jreleaser>
+            <project>
+              <name>inwx-updater</name>
+              <tags>
+                <tag>cli</tag>
+                <tag>inwx</tag>
+              </tags>
+              <maintainers>
+                <maintainer>bmarwell</maintainer>
+              </maintainers>
+              <stereotype>${jreleaser.stereotype}</stereotype>
+              <java>
+                <mainModule>jdyninwx.app</mainModule>
+                <mainClass>${project.java.mainClass}</mainClass>
+                <multiProject>true</multiProject>
+              </java>
+            </project>
+            <distributions>
```

Comment:
This block will be added automatically by the matching javaArchive assembler named `app` because its `exported` property is set to `true`. tags will be inherited from the project. You can safely remove this block.